Move "game logic" to for loop checking user input

gregcorbett opened the pull request:
The example on page 329 of the book uses the layout below for the game loop, so let's follow it.

```
while true:
    for each event:
        check user input
    update game state and screen
```

gregcorbett committed:
Move "game logic" to for loop checking user input

- the example on page 329 of the book uses this layout,
  so let's follow it.

tofu-rocketry commented on PythonLessons/PyGame/pygame-lesson1.py:
```diff
@@ -19,28 +19,29 @@
             pygame.quit()
             sys.exit(0)
+
+        # Fill the screen to prevent any moving shapes leaving a 'trail'
```
This section below is now within the `for...loop`. Was that intended? It doesn't seem to use any `event`s.

norealroots replied:
Using the example Greg said: 
```
while true:
   for each event:
        check user input
    update game state and screen
```

I agree, It doesn't seem like it should be inside the `for...loop`.

gregcorbett replied:
Yes, the original intention was to move it below the `for...loop` not within it. Changed.

norealroots reviewed (changes requested):
It all looks good, but if you could move the
`update game state and screen`
section out from within the for loop first that'd be great!

gregcorbett committed:
Move "game logic" out of for loop

- as this was the original intention of the moving the logic
  in the first place.
